api/v2/checluster_types.go

```diff
@@ -302,6 +302,7 @@ type DevfileRegistry struct {
 	ExternalDevfileRegistries []ExternalDevfileRegistry `json:"externalDevfileRegistries,omitempty"`
 }
 
+// Database component is Deprecated.
```

Comment:
maybe we should add a note that not only it is deprecated but that all the properties set on CR level will be ignored

Reply:
Updated https://github.com/eclipse-che/che-operator/blob/6d55913f45c29347cb48fb2e5d1ae5a89baf5596/api/v2/checluster_types.go#L163